maps/src/MapBinner.cxx

```diff
@@ -263,12 +271,14 @@ MapBinner::BinTimestream(const G3Timestream &det, double weight,
 			(*T)[detpointing[i]] += pcoupling.t*det[i];
 			(*Q)[detpointing[i]] += pcoupling.q*det[i];
 			(*U)[detpointing[i]] += pcoupling.u*det[i];
-			(*W)[detpointing[i]] += mueller;
+			if (W)
+				(*W)[detpointing[i]] += mueller;
```

Comment:
Should we even bother populating the `mueller` variable if it's not being stored?

Reply:
Good point. The compiler should actually optimize that out, but we can adjust.